### ISSUE TITLE
[Feature] Highlight current activity in activity bar more explicitly

### ISSUE THREAD

**Issue by rrmistry** [microsoft/vscode#75435]
I am partially color blind and I have trouble differentiating between the shades of color in the activity bar.

For example, in my current workflow I have the activity bar layout as follows: 
![VS Code Current Tab Highlight](https://user-images.githubusercontent.com/25514145/59449008-41eac980-8dd4-11e9-866c-07ca46c915e2.png)
Might be clear to some of you that it is the icon after the *Source Control* activity (Git Lens) but I cannot tell without serious concentration.

It would be nice to see a more explicit highlight of the current activity.

For example, here is a screen shot from Microsoft Teams:
![Teams Current Tab Highlight](https://user-images.githubusercontent.com/25514145/59449292-c89fa680-8dd4-11e9-99f3-0a61954cb790.png)

Suggested labels: `feature-request`, `accessibility`

Thank you very much for your time and effort. I am deeply grateful.

**Comment by SpectralKH** [comment]
This just got even worse in the 1.37. I'm not colorblind or anything, but from time to time I don't see which one is highlighted either. Using a background color to highlight would definitely be an improvement

**Comment by misolori** [comment]
I'll see if we can add color tokens to achieve this, though I'm not sure whether this would be added as a default to our themes. Would something like below work for you?

<img width="561" alt="image" src="https://user-images.githubusercontent.com/35271042/66340329-c7e9cd80-e8f9-11e9-8255-c28238989a85.png">

**Comment by rrmistry** [comment]
@misolori , yes. The [screenshot](https://user-images.githubusercontent.com/35271042/66340329-c7e9cd80-e8f9-11e9-8255-c28238989a85.png) you provided above would work and significantly improve the user experience for me.

Like you mentioned, it would be nice if this was part of the default theme but not a high priority. Thank you!

**Comment by misolori** [comment]
This will be available in the next Insiders update (likely on Monday). Here's what we changed:

- We tweaked the opacity of `activityBar.inactiveForeground` to make the active element stand out more
- Added `activityBar.activeBorder` which is **on** by default and is inherited by `activityBar.foreground`
- Added `activityBar.activeBackground` which is **off** by default

![gif](https://user-images.githubusercontent.com/35271042/66665267-0be31800-ec03-11e9-9c17-b4b9340bca38.gif)

And if you wanted, you could configure `activityBar.activeBorder`, `activityBar.activeBackground`, and `tab.activeBorder` similarly like:

<img width="640" alt="image" src="https://user-images.githubusercontent.com/35271042/66665369-4056d400-ec03-11e9-80ab-0d9794511c3a.png">
